### PR TITLE
Fix/update examples

### PR DESCRIPTION
**Pull request by ascibisz**
Problem
=======
The Jupyter notebook examples for simularium io are not up to date with the current state of simularium io, and as a result some of the examples are raising errors when they're run!  We want our examples to work, so this ticket was to debug the issues and update the Jupyter notebooks accordingly.
[Link to ticket](https://github.com/simularium/simulariumio/issues/149)

Solution
========
Mostly, the changes include:
* Adding `DisplayType` field in for `DisplayData` since that is now a required parameter
* Fixing a bug I created with part of the CellPack converter I created in a recent PR 😭 but missed because I hadn't tested with any curve data
* Changing the Medyan and Smoldyn examples to use the automatic centering functionality that's being added
* Fixing a few other minor bugs

## Type of change
- Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: examples/Tutorial_smoldyn.ipynb [hunk 1]
```diff
@@ -120,9 +122,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Once your data is shaped like in the `example_data` object, you can use the converter to generate the file at the given path:\n",
-    "\n",
-    "(since this model's coordinates are all positive, use a `TranslateFilter` to center the data in the viewer.)"
```
Comment (ShrimpCryptid) [inline examples/Tutorial_smoldyn.ipynb:125]: Is this behavior now automatically handled as part of the conversion API?

**Review by ShrimpCryptid — APPROVED** [review]
Looks good to me! I'm not super familiar with the Simularium repo so I'll defer to any other opinions :)

File: examples/Tutorial_smoldyn.ipynb (hunk 1 above)
Comment (ShrimpCryptid) [reply examples/Tutorial_smoldyn.ipynb:125]: OOPS found it in the description, nvm!

**Comment by interim17** [comment]
I'm also not sure protocol for testing simio branches, but this overall looks good to me too.

**Review by interim17 — APPROVED** [review]

**Comment by meganrm** [comment]
Nice work. is there any way to have the Jupyter notebook examples auto tested with we run our other tests?

**Comment by ascibisz** [comment]
> Nice work. is there any way to have the Jupyter notebook examples auto tested with we run our other tests?

Good idea! I just used a simple package that'll make sure the Jupyter notebooks execute without raising any errors, and added that to `just test`. Obviously it won't look at the outputted files, but this'll keep us from getting to the state where we're raising errors in the example notebooks again

**Review by meganrm — APPROVED** [review]

**Pull request merged**